src/azure/cli/commands/parameters.py

```diff
@@ -73,6 +73,19 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 def get_enum_choices(enum_type):
     return [x.value for x in enum_type]
 
+class IgnoreAction(argparse.Action): # pylint: disable=too-few-public-methods
+    def __call__(self, parser, namespace, values, option_string=None):
+        raise argparse.ArgumentError(None, 'unrecognized argument: {} {}'.format(
+            option_string, values or ''))
+
+# CONSTANTS
+
+IGNORE_TYPE = CliArgumentType(
```

Comment:
I'm fine with that. I'll address that when I fix item #645 (which I started to do in this PR but decided to keep it focused...)

---
In reply to: [76152207](https://github.com/Azure/azure-cli/pull/744#discussion_r76152207) [](ancestors = 76152207,76151727,76151354)